src/Core/Components/Rating/FluentRating.razor

```diff
@@ -0,0 +1,31 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@inherits FluentInputBase<int>
+
+<FluentInputLabel ForId="@Id" Label="@Label" AriaLabel="@AriaLabel" ChildContent="@LabelTemplate" Required="@Required" />
+
+<FluentStack Id="@Id"
+             Orientation="Orientation.Horizontal"
+             Class="ClassValue"
+             Style="StyleValue"
+             role="radiogroup"
+             @onkeydown="HandleKeyDownAsync"
+             tabindex="@(Disabled ? -1 : 0)"
+             aria-readonly="@(ReadOnly.ToString().ToLower())"
+             HorizontalGap="0">
+    @for (int i = 1; i <= MaxValue; i++)
+    {
+        var currentValue = i;
+        @if (ReadOnly || Disabled)
+        {
+            <FluentIcon Value="@GetIcon(currentValue)" Color="@(Disabled ? Color.Disabled :  IconColor)" />
```

Comment:
We attach great importance to the accessibility of our components. Unfortunately, it's not currently compatible. I think this is mainly because there is no `aria-xxx` on the Icons components.

You can check this by installing https://www.nvaccess.org/download/ and checking the React lib component (which is Accessible) and this new FluentRating component.

If it's too complex in this PR, I suggest adding a note to the documentation and opening a second PR to correct it as soon as possible.

Reply:
do you mean there should be aria-labels for each icon?

Reply:
I think so... but you need to validate using NVDA or the Windows Narrator.

Reply:
I would open a new PR later. Now I don't know how to solve it.

Reply:
No, this accessibility needs to be good from the start.

I believe it is only a matter of adding `aria-label="@currentValue"` to the FluentIcon